sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/rest_client.hpp

```diff
@@ -2159,6 +2170,26 @@ namespace Azure { namespace Storage { namespace Blobs {
         Models::CopyStatus CopyStatus;
       };
     } // namespace _detail
+    /**
+     * @brief Optional, default 'replace'.  Indicates if source tags should be copied or replaced
+     * with the tags specified by x-ms-tags.
+     */
+    class BlobCopySourceTagsMode final {
+    public:
+      BlobCopySourceTagsMode() = default;
+      explicit BlobCopySourceTagsMode(std::string value) : m_value(std::move(value)) {}
```

Comment:
Should only allow copy and replace value. Should be better to blocking initialization from not allowed value